doc/specifications/proposed/named_queries/developer_documentation.md

```diff
@@ -0,0 +1,180 @@
+# Listing Content and Locations
+
+Starting from Platform 2015.07, listing content and locations in a project is
+much easier, using a couple standardized mechanisms.
+
+Predefined, named queries can be defined in any bundle. They can then be used from twig, using a
+[custom controller](#query_controller), or from PHP code using the [QueryType Registry](#fixme). The controller supports
+[template overriding](#template_override), making it easy to use different templates for different calls.
+
+## <a name="query_controller"></a> The query controller
+A new controller is added : `ez_query`.
+
+It has two actions: `locationQuery`, and `contentQuery`.
```

Comment:
After discussing it, I'm still not sold. It's a symfony convention, okay, but it's also about developer experience and consistency. If I could choose, I'd rather use `ez_content:viewContent` rather than `ez_content:viewContentAction`, convention or not convention.